src/sql-parser/tests/testdata/ddl

```diff
@@ -281,7 +281,7 @@ FORMAT BYTES
 ----
```

Comment:
We should add a parse test with keys and maybe one with the `KEYS` keyword but no keys?